opendevin/sandbox/sandbox.py

```diff
@@ -27,8 +27,8 @@
 # How do we make this more flexible?
 RUN_AS_DEVIN = config.get('RUN_AS_DEVIN').lower() != 'false'
 USER_ID = 1000
-if config.get_or_none('SANDBOX_USER_ID') is not None:
-    USER_ID = int(config.get_or_default('SANDBOX_USER_ID', ''))
+if config.get('SANDBOX_USER_ID') is not None:
```

Comment:
Maybe we can bring this key in config.py?

Reply:
> config.py

config.toml?